### ISSUE TITLE
TiltSeries display error?

### ISSUE THREAD

**Issue by azazellochg** [scipion-em/scipion-em-tomo#129]
I have imported 5 TS movies (41 tilt each). In the viewer it says 0 items, while I assume it should be saying 41 items

![image](https://user-images.githubusercontent.com/6952870/123090406-fe45e400-d41f-11eb-9f1d-2e97a92940eb.png)

**Comment by azazellochg** [comment]
I verified that in TiltSeriesBase the _size is incremented by Set class
```
    def append(self, tiltImage):
        tiltImage.setTsId(self.getTsId())
        data.SetOfImages.append(self, tiltImage)
```
However, self.getSize() from `__str__` still returns 0.  @pconesa Can this be because of ignoreAttrs=['_mapperPath', '_size']?

**Comment by pconesa** [comment]
We had a look recently to this. The thing is that for historical/architectural reasons, we need to 1st, persist the TiltSeries/TiltSeriesM empty, therefore it ends up persisting its attributes with  size = 0. Then TiltImages are added, size incremented, but never persisted again.

I think we have fixed this for TiltSeries, but have missed TiltSeriesM?

Ok, we did something at set level, but might need to fix this at TiltSeries level:
https://github.com/scipion-em/scipion-em-tomo/pull/126/files
